### PR TITLE
allow name override on operator without changing name of resources

### PR DESCRIPTION
**Pull request by josh-pritchard**
Allows me to generate templates in the new chart like this!
image

BOT NOTES: 
resolves https://github.com/solo-io/skv2/issues/432

### CODE REVIEW

**Review by t-edris — COMMENTED** [review]
Would it be worth it to make a test like [this one](https://github.com/solo-io/skv2/blob/master/codegen/cmd_test.go#L378) where we ensure that the `ValuesFileNameOverride` is used properly? Other than that, this looks like a simple enough change

**Comment by josh-pritchard** [comment]
yep good call, just added a test

File: codegen/cmd_test.go [hunk 1]
```diff
@@ -243,6 +243,94 @@ var _ = Describe("Cmd", func() {
 		Expect(envMapField.HeadComment).To(Equal("# Environment variables for the container. For more info, see the [Kubernetes\n# documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#envvarsource-v1-core)."))
 	})
 
+	FIt("generates from templates using a name override", func() {
```
Comment (marcogschmidt) [inline codegen/cmd_test.go:246]: Unfocus

**Review by marcogschmidt — APPROVED** [review]
LGTM

**Review by t-edris — APPROVED** [review]

**Pull request merged**